src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $card-width;
+  padding: $card-width * 0.01 * 16 $card-width * 0.01 * 8 $card-width * 0.01 * 8;
```

Comment:
The padding shorthand property is being used with calculations. Ensure that the calculations are correct and that they achieve the desired padding values. Consider using variables for clarity if the same calculation is used multiple times.